eslint/eslint-config-local/profile/_common.js

```diff
@@ -0,0 +1,133 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
```

Comment:
Why re-declare all this stuff? I know internally we have a change that should allow for extending from configs in a package with any name in order to extend from `@rushstack/eslint-config`. Maybe we should bring that in with this PR to reduce duplication?